app/views/performance_dashboards/overview/_universe_selection.haml

```diff
@@ -4,6 +4,7 @@
     - next unless section
     -# - section_url = polymorphic_path(@report.filter_path_array, *(link_params.except(:action).merge(filter_section_id: section.id)))
     - section.controls.each do |control|
+      - next if control.id == 'inactivity_days'
```

Comment:
hide it from the "your universe" pills
![Screen Shot 2022-05-03 at 4 52 51 PM](https://user-images.githubusercontent.com/5333982/166563975-8515db38-94cf-472b-8b17-ba49a9031ab8.png)